commit 76535fcda32c3af3e02876afb80c6b16242af9b3 (refs/changes/24/1124/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-14T16:13:06-08:00 (2 years, 10 months ago)
    
    joyent/node-smartdc-auth#11 privateKeySigner doesn't allow opts.key to be a buffer
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/lib/index.js b/lib/index.js
index a9bedb3..0181835 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -80,7 +80,6 @@ function privateKeySigner(options) {
 		throw (new Error('options.key (a String or Buffer) is ' +
 		    'required'));
 	}
-	mod_assert.string(options.key, 'options.key');
 	mod_assert.string(options.user, 'options.user');
 	mod_assert.optionalString(options.subuser, 'options.subuser');
 	mod_assert.optionalString(options.keyId, 'options.keyId');
diff --git a/package.json b/package.json
index e37ff7b..b58f8b4 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "smartdc-auth",
     "description": "Joyent Authentication Library",
-    "version": "2.5.2",
+    "version": "2.5.3",
     "author": "Joyent (joyent.com)",
     "contributors": [
         "Mark Cavage",
